wca-guidelines.md

```diff
@@ -156,7 +158,7 @@ To be more informative, each Guideline is classified using one of the following
 - A1a2+++) [CLARIFICATION] In case of a cumulative time limit, the result of each remaining attempt in the round is recorded as DNS (Did Not Start) after a competitor reaches the time limit.
 - A1a2++++) [ADDITION] If a cumulative time limit for a single event or a combination of events is enforced, such events don't have a regular time limit per solve.
 - A1a2+++++) [CLARIFICATION] Events where a time limit per attempt is defined by the Regulations (3x3x3 Fewest Moves and 3x3x3 Multi-Blind) must not have a cumulative time limit.
-- A1a3+) [REMINDER] The organization team and the WCA Delegate must be mindful that time limits influence the strategies of the competitors (e.g. rushing the first two attempts in hopes of meeting a cutoff in a combined round), and that changing time limits after the start of the round can disadvantage some competitors unfairly.
+- A1a3+) [REMINDER] The organization team and the WCA Delegate must be mindful that time limits influence the strategies of the competitors (e.g. rushing the first two attempts in hopes of meeting a cutoff), and that changing time limits after the start of the round can disadvantage some competitors unfairly.
```

Comment:
I didn't make a copy of this for cutoffs, but I wouldn't be opposed to it either.
(If we added something like this for cutoffs, I think we should make it more concise and point at A1a3+ so that the spirit of the guideline is encapsulated in one place.)

Reply:
Hmm, it's weird to me that A1a3+) starts speaking of "time limits" and the example that lists talks about "cutoffs" :P I never thought about that before, but now that I did, I don't like the wording of this Guideline.